lib/host/wasi_nn/CMakeLists.txt

```diff
@@ -5,8 +5,33 @@ wasmedge_add_library(wasmedgeHostModuleWasiNN
   wasinnmodule.cpp
 )
 
-target_link_libraries(wasmedgeHostModuleWasiNN
+list(APPEND INCLUDE_DIRS "${Boost_INCLUDE_DIRS}")
+list(APPEND INCLUDE_DIRS "${PROJECT_SOURCE_DIR}/include")
+list(APPEND INCLUDES_DIRS "${PROJECT_SOURCE_DIR}/thirdparty")
+
+
+list(APPEND LIB_DIRS "wasmedgeSystem")
```

Comment:
It's better to use `target_link_libraries` and `target_include_directories`. Please refer to the other `CMakeLists` files.